src/Service/FileUploader.php

```diff
@@ -46,7 +43,7 @@ public function uploadSponsor(Request $request)
     {
         $file = $this->getFileFromRequest($request);
 
```

Comment:
Maybe we should verify filetype?

(Suggestion should probably be extracted to separate function as it is already used in `uploadReceipt()`)

```suggestion

        $mimeType = $file->getMimeType();
        $fileType = explode('/', $mimeType)[0];
        if ($fileType === 'image') {
            return $this->uploadFile($file, $this->receiptFolder);
        } else {
            throw new BadRequestHttpException('Filtypen må være et bilde.');
        }
```